### ISSUE TITLE
Spitz AX3000 - no modems

### ISSUE THREAD

**Issue by modemmike** [TalalMash/Unofficial-Speedify-Installer-For-OpenWrt#13]
I broke down and spent the cash for this amazing router... just 1 gltich for me at the moment.

Any thoughts on why modems are not being picked up in Speedify?

![image](https://github.com/TalalMash/Unofficial-Speedify-Installer-For-OpenWrt/assets/5247152/50343273-ca10-48dd-856e-2f0580b10d1b)
![image](https://github.com/TalalMash/Unofficial-Speedify-Installer-For-OpenWrt/assets/5247152/8a1177e5-0d60-4ccf-bb14-208e80ed3f12)
![image](https://github.com/TalalMash/Unofficial-Speedify-Installer-For-OpenWrt/assets/5247152/85778d2b-e96e-410e-aeae-7e26d51f81b4)

Repeater, ethernet and USB tethering are all seen, just not the modem.

**Comment by TalalMash** [comment]
Contact Speedify, I think they're about to release a router version, the unofficial method is using the desktop version, maybe there is a new restriction? Try downgrading in the Speedify menu to an older version to check. Generate logs button will show Speedify logs.
Check the load balancing options in GL.iNet admin interface, maybe the default gateway isn't being provided, can be checked in terminal with `ip route`.

**Comment by TalalMash** [comment]
Check the load balancing options in GL.iNet admin interface, maybe the default gateway isn't being provided, can be checked in terminal with ip route. It's likely that the modem is set as a failover mode in GL.iNet.

**Comment by modemmike** [comment]
Yep, I had to switch to Load Balance, make all others 0, and then cellular set to 3. Whether or not that is correct, IDK but Speedify saw all 4 connections that way. My goal is to livestream (mobile). I tested by pulling a connection which only caused a tiny blip. Pretty cool.

This router is worth $400 to me now :-) it is built well, nice options for cell modem configuration etc.

Thank you @TalalMash
